src/widgets/courses/ui/courses.test.tsx

```diff
@@ -0,0 +1,81 @@
+import { screen } from '@testing-library/react';
+import { Mock, describe, expect, it, vi } from 'vitest';
+
+import { Courses } from './courses';
```

Comment:
Not sure we use the blank line in other places

Reply:
Deleted